files/image_config/constants/constants.yml

```diff
@@ -33,7 +33,7 @@ constants:
           - "deny 0::/0 le 59"
           - "deny 0::/0 ge 65"
     bbr:
-      enabled: true
+      enabled: false
```

Comment:
On second thought, changing enabled to false would cause BBRManager drop all set requests according to https://github.com/sonic-net/sonic-buildimage/blob/4e78b11c96192d3c1e4332d5bca9c415ad7c1c9d/src/sonic-bgpcfgd/bgpcfgd/managers_bbr.py#L31.

Reply:
There is a bug in this code which gives the following behaviour.
IF BBR is enabled in the constants.yml and the configuration of BGP_BBR|all etnry is disabled, The BBR remains disabled until the config reload. At SONIC reload, the managers_bbr reads the constants,yml and this configuration takes precedence over config_db BGP_BBR|all toggle. As a result the deivces comes up with BBR enabled. For overlay ECMP, this causes T1s to  learn  routes via BGP which inturn causes traffic blackholing of ECMP routes. ADO for the bug. 28302790
Since this feature is not being actively used, we are disabling it by default until this bug is fixed.